perfeed/models/pr_summary.py

```diff
@@ -75,5 +75,5 @@ class PRSummaryMetadata(BaseModel):
     llm_provider: str
     model: str
     pr_created_at: str
-    pr_merged_at: str
+    pr_merged_at: Optional[str] = None
```

Comment:
`pr_merged_at: str | None = None` would work without additional import for `Optional`

Reply:
i will keep as is just for the consistency between this and PullRequest.merged_at